telegram-bot/src/test/java/pro/sky/telegrambot/NotificationServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package pro.sky.telegrambot;
+
+import com.pengrad.telegrambot.TelegramBot;
+import com.pengrad.telegrambot.model.Chat;
+import com.pengrad.telegrambot.request.SendMessage;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import pro.sky.telegrambot.model.Notifications;
+import pro.sky.telegrambot.repositories.NotificationsRepository;
+import pro.sky.telegrambot.service.NotificationServiceImpl;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static pro.sky.telegrambot.СonstantsTestBot.*;
+
+@ExtendWith(MockitoExtension.class)
+public class NotificationServiceImplTest {
+    @Mock
+    private NotificationsRepository notificationsRepositoryMock;
+
+    @Autowired
+    private TelegramBot telegramBot;
+    @InjectMocks
+    private NotificationServiceImpl out;
+
+    // эти методы допишу, если удастся разобраться с вопросом по коснтанте Update -
+    // см комметарий в кл СonstantsTestBot
+
+//    @Test// НЕ ЗАВЕРШЕН, нужно дописать логику
+//    public void shouldReturnGreeting() {
+//        when(updateMock.message().chat().id()).thenReturn(1L);
+//        when(out.getNameOfUser(updateMock)).thenReturn("User 1");
+//        SendResponse resultGreeting = out.greeting(updateMock);
+//        assertNotNull(resultGreeting);
+//    }
+
+    //        @Test
+//    public void shouldReturnNameOfUser() {
```

Comment:
в getNameOfUser в качестве параметров можно сразу отдавать строки, если хочется протестировать эту логику

Reply:
не совсемпонял, что ты имеешь ввиду... я в параметры метода public String getNameOfUser(Update update) подаю тот же класс Update и из него достаю имя... опять же, пытаюсь его замокать сталкиваюсь с той же проблеммой 
Cannot invoke "com.pengrad.telegrambot.model.Message.from()" because the return value of "com.pengrad.telegrambot.model.Update.message()" is null